openapi.yml

```diff
@@ -3384,6 +3387,87 @@ paths:
+      tags:
+        - Capacities
+      responses:
+        '200':
+          description: 'Successful response returns the current number of signins, expected invites and expected invites for the next N number of hours.'
+          content:
+            application/json:
+              schema:
+                $ref: ./models/CapacityResponse.v1.yaml
+              examples:
+                default:
+                  value:
+                    signins_count: 189
+                    invites_expected_next_hour: 1
+                    invites_by_hour:
+                      - range_start: '2020-07-17T00:59:59.999Z'
+                        range_end: '2020-07-17T01:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T01:59:59.999Z'
+                        range_end: '2020-07-17T02:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T02:59:59.999Z'
+                        range_end: '2020-07-17T03:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T03:59:59.999Z'
+                        range_end: '2020-07-17T04:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T04:59:59.999Z'
+                        range_end: '2020-07-17T05:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T05:59:59.999Z'
+                        range_end: '2020-07-17T06:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T06:59:59.999Z'
+                        range_end: '2020-07-17T07:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T07:59:59.999Z'
+                        range_end: '2020-07-17T08:59:59.999Z'
+                        expected_visitors: 1
+        '400':
+          description: A generic error
+          content:
+            application/json:
+              schema:
+                $ref: ./models/ErrorsList.v1.yaml
+              examples:
+                default:
+                  value:
+                    errors:
+                      - domain: some text
+                        attribute: some text
+                        code: some text
+                        message: some text
+                      - domain: some text
+                        attribute: some text
+                        code: some text
+                        message: some text
+        '401':
+          description: "You don't have permission to view the details of the location's capacity."
+        '403':
+          description: You do not have permission for this action
+      operationId: getLocationCapacity
+      description: Gets the details of the capacity in a location.
+      parameters:
+        - schema:
+            type: integer
```

Comment:
We'll probably need to add a timestamp as the start point, as well. EG, if somebody is looking at the planned capacity at 2am, they likely want to see the next morning. The default can be the current timestamp.

Reply:
But if we are gonna pass the timestamp as a query params, won't we fall into the same issue you specified above?(except that local_time is now renamed as timestamp)

Your comment for reference:
2222-07-16T17:05:08-07:00
Will we assume that the local time is 202 years in the future?

In the response we have an input named "current_signins", won't this attribute be confusing for the user? 
Current signins are people signed in a location, no matter when he signed in, we just check whether he is signed in.

So when I pass a timestamp, I can calculate the future invites for that time, but I am gonna give back the current_signins no matter what the timestamp and people are gonna assume that there will be that many signins at the timestamp that they have passed in as an input.